catalogue/webapp/components/WorkLink/index.tsx

```diff
@@ -15,15 +15,16 @@ type Props = {
   id: string;
   source: WorkLinkSource;
   resultPosition?: number;
+  children: ReactNode;
```

Comment:
We use `PropsWithChildren` in several places in the codebase – feels like the "right" way to do it to me, rather than typing the `children` ourselves like you've done here – is there a reason one way would be better than another? I feel like the `PropsWithChildren` feels future-proof

Reply:
> We use `PropsWithChildren` in several places in the codebase – feels like the "right" way to do it to me, rather than typing the `children` ourselves like you've done here – is there a reason one way would be better than another?

I'm not sure; it feels like an FC vs FunctionComponent thing to me. We should pick a style and then use it consistently (and I don't mind which one).

Do you know if @AMBacelar has picked a particular approach? I know he's had to add this a bunch of times for the Next 13 upgrade.

Reply:
We discussed briefly yesterday. I think its slightly different from the FC/FunctionComponent thing because they're two things that React give us  (one of which is an alias for the other). This is one thing that React gives us or one thing that we write ourselves.

Reply:
Fixed here, and there's a ticket to make this consistent everywhere (once Adi has finished his changes): https://github.com/wellcomecollection/wellcomecollection.org/issues/9437